cli/src/ui/App.tsx

```diff
@@ -72,7 +79,7 @@ const App: FC<AppProps> = ({ cli }) => {
     case "agents-mcp":
       return <AgentsMCP port={flags.port} sId={flags.sId} />;
     case "chat":
-      return <Chat sId={flags.sId?.[0]} agentSearch={flags.agent} message={flags.message} conversationId={flags.conversationId} />;
+      return <Chat sId={flags.sId?.[0]} agentSearch={flags.agent} message={flags.message} conversationId={flags.conversationId} messageId={flags.messageId} details={flags.details} />;
```

Comment:
we should avoid sending the non interactive message via the chat component, check here with the flags if the message should be non interactive, and send it to a <NonInteractive> component, or similar.

not mandatory in that PR, but would be nice to have i think